docs/sdks/server-sdks/python/announcing-python-4-0.md

```diff
@@ -0,0 +1,182 @@
+# Announcing Python SDK 4.0
+
+We’re excited to announce the release of Python SDK 4.0!
+
+This release signifies a big milestone for the Python SDK.
+It’s a complete rewrite of the SDK using a shared Eppo core library written in Rust.
+
+Using a shared core library allows us to centralize our efforts and provide better performance, stability, and deliver new features faster.
+
+## Better performance
+
+Thanks to Rust’s natural speed, the assignment evaluation performance improved 2–5x depending on the feature flag configuration.
+You may expect even better performance in the future as we continue to optimize the core library.
+
+Additionally, configuration update now happens in a background thread without ever holding Global Interpreter Lock (GIL), so it runs in parallel with your Python threads without slowing them down.
+
```

Comment:
I wonder if this is a good spot to put the default polling interval has been changed from 300 seconds to 30

Reply:
Thanks for the reminder! Aaron is referring to this: 

https://github.com/Eppo-exp/python-sdk/blob/main/eppo_client/constants.py